src/mngt/ocf_mngt_cache.c

```diff
@@ -1972,11 +1975,19 @@ static void ocf_mngt_cache_stop_finish(ocf_pipeline_t pipeline,
 				context->cache_name);
 	}
 
-	context->cmpl(cache, context->priv,
-			error ?: context->cache_write_error);
+	/*
+	 * FIXME This is a temporary workaround to prevent dereferencing pipeline
```

Comment:
I propose a different message here:

FIXME: Destroying pipeline before completing management operation is a temporary workaround for insufficient object lifetime management in pyocf. Context must not be referenced after destroying pipeline as this is typically freed upon pipeline destroy.